Dockerfile

```diff
@@ -32,23 +32,11 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM buildpack-deps:stretch AS base
+FROM golang:1.9.4 AS base
 # allow replacing httpredir or deb mirror
 ARG APT_MIRROR=deb.debian.org
 RUN sed -ri "s/(httpredir|deb).debian.org/$APT_MIRROR/g" /etc/apt/sources.list
 
-
-FROM base AS golang
-# IMPORTANT: If the version of Go is updated, the Windows to Linux CI machines
-#            will need updating, to avoid errors. Ping #docker-maintainers on IRC
-#            with a heads-up.
-# IMPORTANT: When updating this please note that stdlib archive/tar pkg is vendored
-ENV GO_VERSION 1.9.4
```

Comment:
Actually, just realised that we use the `GO_VERSION` env-var as well in the scripts to generate the "builder" images; https://github.com/moby/moby/blob/4bdb6ed9b58d641176a601cfe92d496038c5a557/contrib/builder/deb/ppc64le/generate.sh#L89

Reply:
ohhh :sweat: ok I think I have an idea of what to do :angel: